data/human/human missions.txt

```diff
@@ -5547,3 +5547,128 @@ mission "Quicksilver Mixup 1"
 			
 
 event "quicksilver mixup 2 available"
+
+
+
+mission "Argosy Hijacking (Trigger)"
+	invisible
+	source
+		government "Syndicate"
+	to offer
+		random < 10
+	on offer
+		event "argosy hijacking trigger"
+		fail
+
+event "argosy hijacking trigger"
+
+mission "Argosy Hijacking"
+	landing
+	name "Stop Ship Hijacking"
+	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
+	source
+		government "Syndicate"
+		not attributes "station"
+		near "Alcyone" 4
+		near "Almach" 3
+		near "Gienah" 3
+	to offer
+		has "event: argosy hijacking trigger"
+	substitutions
+		"<argosy hijacking variable>" "destroy"
+		"<argosy hijacking variable>" "board"
+			has "boarded hijacked argosy"
+	on offer
+		conversation
+			`You're signing the standard Syndicate landing papers with an officer on <origin>. "Everything seems to be in order. And of course, we'll have your ship refueled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`
+			`	Before he can make it there, you hear a commotion by the <npc model> on the landing pad next to you.`
+			choice
+				`	(Turn around to see what's going on.)`
+					goto turn
+				`	(Ignore the commotion.)`
+					goto ignore
+			label turn
+			`	Both of you turn your head to see what's happening, but as you do a deafening blare of warning sounds through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
```

Comment:
```
			`	Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
```
I feel like this could be rephrased so there's less comma. It's a little jarring to read.